samples/discoverNatBehavior.c

```diff
@@ -2,12 +2,12 @@
 
 #define NETWORK_INTERFACE_NAME_PARAM "-i"
 #define STUN_HOSTNAME_PARAM          "-s"
-#define DEFAULT_STUN_HOST            "stun:stun.sipgate.net:3478"
+#define DEFAULT_STUN_HOST            "stun:stun.kinesisvideo.us-west-2.amazonaws.com:443"
```

Comment:
If we are using our STUN host anyways, I think we should generate this string region based. Use the env that is set up for region. If not set, we can default to us-west-2 with a log message

Reply:
Done!